docs/topics/proc_deploying-the-circuit-breaker-booster-using-the-oc-cli-client.adoc

```diff
@@ -1,21 +1,20 @@
-[[build_and_deploy_booster]]
-= Building and Deploying the Booster
-
-ifndef::circuit-breaker-nodejs[]
-include::booster-deploy-openshift-online.adoc[leveloffset=+1]
-endif::circuit-breaker-nodejs[]		
 
+[#deploying-the-booster-using-the-oc-cli-client_{context}]
```

Comment:
May I ask you to include the booster name in the anchor ID? Two reasons:

* It makes the anchor ID follow the file name more closely.
* It prevents multiple IDs of such modules used in the same context to collide.

In this case, it would be `[#deploying-the-circuit-breaker-booster-using-the-oc-cli-client_{context}]`.

Valid for all such occurrences.